<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRetypePassword">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Password: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUsername">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRetypePassword">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="monthGroupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth</string>
   </property>
   <widget class="QComboBox" name="monthComboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="dayLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yearLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="genderGroupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="maleRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="femaleRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="AccountTypeGroupBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="userRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="adminRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>69</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="genresGroupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Genres</string>
   </property>
   <widget class="QCheckBox" name="actionCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ComedyCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="romanceCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="otherCheckBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="horrorCheckBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dramaCheckBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="errorAlreadyExisting">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* Already Existing</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorAllFieldMustBeFilled">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>470</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* All fields must be filled</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorAgesLessThanTweve">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* Age is less than 12</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorNotMatching">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* Not Matching</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
